src/web/mjs/connectors/Funbe.mjs

```diff
@@ -0,0 +1,47 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class Funbe extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'funbe';
+        super.label = 'Funbe';
+        this.tags = [ 'manga', 'korean', 'hentai' ];
+        this.url = 'https://funbe.care';
+        this.language = 'ko';
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/망가', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.list-container div.section-item-title a');
+        return data.map(link => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(link.href, this.url).replace('hakuneko://cache/', '/'),
```

Comment:
It seems there is an issue with stripping of the connector origin.
This should not be handled on the connector itself, instead it should be handled in the [base class](https://github.com/manga-download/hakuneko/blob/7fa114a621bc2a737e59f360dff4aa0a0c1bdce5/src/web/mjs/engine/Connector.mjs#L381).

This seems to be a regression, as this worked before without any problems... maybe the last electron update caused this.

Reply:
Try to not spread the href attribute manually, instead let the method do it:
`this.getRootRelativeOrAbsoluteLink(link, this.url)`

Reply:
I tried that first, but it will return links starting with `hakuneko://cache/`. My test results are that `this.createDOM(data)` is fine but `this.createDOM(data).querySelectorAll('body a')` isn't.

Reply:
I checked out your branch and it worked fine for me, seems to be an isolated issue...

![image](https://user-images.githubusercontent.com/27136773/81501485-e4f3de00-92d8-11ea-8a0c-c4334007d438.png)

Reply:
Fyi. I'm still trying to resolve this, but something caches the connector itself and changes are not reflected to the running connector (allthough the right contents are shown in the devlopment source view of chrome).

Reply:
The function call itself already looks wrong on my end:
![image](https://user-images.githubusercontent.com/18627623/82784092-1daac000-9e60-11ea-94ab-14153535d2a8.png)
And I already see this directly after `fetchDOM`:
![image](https://user-images.githubusercontent.com/18627623/82784285-7aa67600-9e60-11ea-817b-f376ddd66c14.png)
@ronny1982 can you please test if everything works without the replace at your end? I'll remove it then.